package.json

```diff
@@ -109,6 +109,6 @@
     "xml-js": "^1.6.11"
   },
   "engines": {
-    "node": ">=10 <=15"
+    "node": ">=10 <=16"
```

Comment:
we should test version 10, 11, 12, 13 and 14 from which version on we support building web. I'm also fine to keep it and bring back core-js as a babel step what i did to make my performance pr buildable in ci with node 14

Reply:
@kulmann tested node 10 and it worked, so i'm fine to bump node for ci and keep version 10 in package.json